Aplib.Core.Tests/Belief/BeliefSetTests.cs

```diff
@@ -134,6 +134,9 @@ private class SimpleBelief : IBelief
         /// </summary>
         public bool Updated { get; private set; } = false;
 
+        /// <inheritdoc />
+        public IMetadata Metadata { get; } = new Metadata("SimpleBelief");
```

Comment:
This does not match the current standards, and should be assigned in a constructor instead.